examples/finetune/README.md

```diff
@@ -0,0 +1,47 @@
+# finetune
+
+Basic usage instructions:
+
+```bash
+# get training data
+wget https://raw.githubusercontent.com/brunoklein99/deep-learning-notes/master/shakespeare.txt
+
+# finetune LORA adapter
+./bin/finetune \
+        --model-base open-llama-3b-v2-q8_0.gguf \
+        --checkpoint-in  chk-lora-open-llama-3b-v2-q8_0-shakespeare-LATEST.gguf \
+        --checkpoint-out chk-lora-open-llama-3b-v2-q8_0-shakespeare-ITERATION.gguf \
+        --model-out lora-open-llama-3b-v2-q8_0-shakespeare-ITERATION.bin \
```

Comment:
There is no `--model-out` argument

Reply:
Ah yes, it is '--lora-out'. Fixed the readme.